### ISSUE TITLE
Cellphone Checkmark

### ISSUE THREAD

**Issue by peterbork** [Templarian/MaterialDesign#5874]
I have:

- [x] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [x] Searched open issues to make sure there isn't a request for this icon.
- [x] Followed the [material guidelines](https://material.io/design/iconography/system-icons.html).

## Preview
(http://dev.materialdesignicons.com/contribute/github).
![image](https://user-images.githubusercontent.com/4498824/109513319-4160fd80-7aa5-11eb-8145-b52ebba56407.png)

## Zip Download
[cellphone-checkmark.zip](https://github.com/Templarian/MaterialDesign/files/6062387/cellphone-checkmark.zip)

**Comment by Templarian** [comment]
Probably going to prefer the standard round cutout check style in the bottom right. One of the contributors can probably take care of this with the template.

https://dev.materialdesignicons.com/contribute/icon-naming#overlay

**Comment by peterbork** [comment]
Okay @Templarian, i just wanted something to go with `cellphone-erase`.
![cellphone-erase](https://user-images.githubusercontent.com/4498824/109613765-e9240d00-7b31-11eb-8471-70e05f32a5fd.png)

**Comment by goyney** [comment]
Yeah, that icon should also be updated to use our normal modifier. 😄

**Comment by martijnvegmond-cw** [comment]
In the attachment I created all overlay versions in SVG.
Don't know if it they're ok, but maybe it could be checked.

[MDI_cellphone-overlay.zip](https://github.com/Templarian/MaterialDesign/files/6596655/MDI_cellphone-overlay.zip)
